docs/advanced/lambda.md

```diff
@@ -23,7 +23,7 @@ type Event struct {
 func main() {
     lambda.Start(func(ctx context.Context, event Event) (string, error) {
         return fmt.Sprintf("Hello %s!", event.Name), nil
-    }
+    })
```

Comment:
actually the correct is

```
func main() {
	lambda.Start(func(ctx context.Context, event Event) (string, error) {
		return fmt.Sprintf("Hello %s!", event.Name), nil
	})
}
```

Reply:
What do you mean? I did fix it already. Do you mean replacing 4 spaces into tabs?

Reply:
I don't feel strongly about the tabs vs spaces, folks who use the code will have it gofmt'ed.